Ukoly/DU_9/class.py

```diff
@@ -0,0 +1,64 @@
+class band_member:
+    def __init__(self, name):
+        self.name = name
+
+    def play(self):
+        print(f'{self.name} plays on {self.instrument}')
```

Comment:
Čo ak `self.instrument` nie je definované (je rovné `None`)? Nebolo by to dobré nejako ošetriť? Ideálny spôsob, ako napísať takúto abstraktnú triedu (to je taká trieda, že sama o sebe sa nepoužíva na instanciovanie, ale ako nejaký spoločný rodič ďalších pod-tried - ono to má svoj význam v polymorfizme), tak sa dáva na časť, ktoré závisia od rodičov vyvolanie výnimky `NotImplementedError`. V skratke ak neexistuje `self.instrument`, tak vyhodíš výnimku `NotImplementedError`.